packages/core/src/accordion/accordion-panel.element.scss

```diff
@@ -22,8 +20,6 @@
 
 .accordion-content {
   overflow-y: hidden;
-  box-shadow: 0 calc(-1 * var(--border-width)) 0 0 var(--border-color);
-  margin-top: var(--border-width); // this helps with a browser issue that hides the top border above in some cases
```

Comment:
This CSS was causing a thin white line to appear in collapsed header background. We use a box-shadow on the accordion content to draw a border between the header and content. Since this is only applicable when the panel is expanded, moving this code snippet to expanded selector (shown in lines 51-55) fixes this visual bug.